rbac/server/api/users.py

```diff
@@ -122,10 +125,13 @@
     )
 
     # Submit transaction and wait for complete
-    await utils.send(
+    sawtooth_response = await utils.send(
         request.app.config.VAL_CONN, batch_list, request.app.config.TIMEOUT
     )
 
+    if not sawtooth_response:
```

Comment:
This should raise an error instead of just logging the error because it will continue to create an auth entry for the user that was not properly created.